pages/linux/ncdu.md

```diff
@@ -0,0 +1,19 @@
+# ncdu
+
+> Disk usage analyzer with an ncurses interface.
+
+- Analyze current working directory:
+
+`ncdu`
+
+- Analyze given directory:
+
+`ncdu directory/`
```

Comment:
Could `directory` be `{{path/to/directory}}` please?

Reply:
done